website/docs/r/serviceendpoint_generic_git.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_serviceendpoint_generic_git"
+description: |-
+  Manages a generic service endpoint within Azure DevOps, which can be used to authenticate to any external git service
+  using basic authentication via a username and password.
+---
+
+# azuredevops_serviceendpoint_generic_git
+
+Manages a generic service endpoint within Azure DevOps, which can be used to authenticate to any external git service
+using basic authentication via a username and password. This is mostly useful for importing private git repositories.
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "project" {
+  name       = "Sample Project"
+  visibility         = "private"
+  version_control    = "Git"
+  work_item_template = "Agile"
+}
+
+resource "azuredevops_serviceendpoint_generic_git" "serviceendpoint" {
+  project_id            = azuredevops_project.project.id
+  repository_url        = "https://dev.azure.com/org/project/_git/repository"
+  username              = "username"
+  password              = "password"
+  service_endpoint_name = "Sample Generic Git"
+  description           = "Managed by Terraform"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `project_id` - (Required) The project ID or project name to associate with the service endpoint.
+- `service_endpoint_name` - (Required) The name of the service endpoint.
+- `repository_url` - (Required) The URL of the repository associated with the service endpoint.
+- `username` - (Optional) The username used to authenticate to the git repository using basic authentication.
+- `password` - (Optional) The password or token key used to authenticate to the git repository using basic authentication.
+- `description` - (Optional) The Service Endpoint description. Defaults to `Managed by Terraform`.
```

Comment:
`enable_pipelines_access` not documneted

Reply:
:feelsgood:

Reply:
fixed, and rebased